package.json

```diff
@@ -15,6 +15,7 @@
     "make": "electron-forge make",
     "publish": "electron-forge publish",
     "lint": "eslint --ext .ts,.tsx .",
+    "lint-ci": "eslint --max-warnings 0 --ext .ts,.tsx .",
```

Comment:
What does this do?

Reply:
it's used in the CI [here](https://github.com/diffix/desktop/pull/255/files#diff-2ace50007d05636bdbb4ee32251052be22a3bcb9f01108e2cbb7a2df05abc36eR29) to catch missed linting warnings like [this](https://github.com/diffix/desktop/pull/253#discussion_r749469104)

Reply:
I don't think a separate command is warranted here. The original one should treat warnings as errors.

Reply:
if this doesn't break anyones existing workflow/integration, I agree to have just the strict `lint`